src/engine/blob-extension.md

```diff
@@ -5,6 +5,27 @@ This extension is backwards-compatible, but not part of the initial Engine API.
 
 ## Structures
 
+### ExecutionPayloadV2
+
+This structure is extended to include a single field: `excessDataGas`.
+
+- `parentHash`: `DATA`, 32 Bytes
+- `feeRecipient`:  `DATA`, 20 Bytes
+- `stateRoot`: `DATA`, 32 Bytes
+- `receiptsRoot`: `DATA`, 32 Bytes
+- `logsBloom`: `DATA`, 256 Bytes
+- `prevRandao`: `DATA`, 32 Bytes
+- `blockNumber`: `QUANTITY`, 64 Bits
+- `gasLimit`: `QUANTITY`, 64 Bits
+- `gasUsed`: `QUANTITY`, 64 Bits
+- `timestamp`: `QUANTITY`, 64 Bits
+- `extraData`: `DATA`, 0 to 32 Bytes
+- `baseFeePerGas`: `QUANTITY`, 256 Bits
+- `blockHash`: `DATA`, 32 Bytes
+- `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
+- `withdrawals`: `Array of WithdrawalV1` - Array of withdrawals, each object is an `OBJECT` containing the fields of a `WithdrawalV1` structure.
+- `excessDataGas`: `QUANTITY`, 256 bits
```

Comment:
Could we move this field between `baseFeePerGas` and `blockHash` to match the field ordering in the consensus spec objects: https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/beacon-chain.md#executionpayload

Reply:
yup. done